docs/content/docs/dev/table/sql/show.md

```diff
@@ -504,6 +561,38 @@ SHOW CURRENT CATALOG
 
 Show current catalog.
 
+## SHOW CREATE CATALOG
+
+```sql
+SHOW CREATE CATALOG catalog_name
+```
+
+Show creation statement for an existing catalog.
+
+The output includes the catalog's name and relevant properties, which allows you to easily reuse the created catalogs.
```

Comment:
Nit: I think the phrase "easily resuse the created catalogs" doesn't quite capture the intended meaning. A more accurate description would be "gain an intuitive understanding of the underlying catalog's metadata."

Reply:
Thank you for pointing it out, I have updated as you said :)